src/screens/HomeScreen.js

```diff
@@ -24,6 +24,130 @@ import { ScreenName } from '../types';
 
 const { MATOMO_TRACKING, ROOT_ROUTE_NAMES } = consts;
 
+const renderItem = ({ item }) => {
+  const {
+    buttonTitle,
+    categoriesNews,
+    fetchPolicy,
+    showData,
+    navigate,
+    navigation,
+    query,
+    queryVariables,
+    showVolunteerEvents,
+    title
+  } = item;
+
+  const NAVIGATION = {
+    CATEGORIES_INDEX: {
+      name: ScreenName.Index,
+      params: {
+        title,
+        query: QUERY_TYPES.CATEGORIES,
+        queryVariables: {},
+        rootRouteName: ROOT_ROUTE_NAMES.POINTS_OF_INTEREST_AND_TOURS
+      }
+    },
+    EVENT_RECORDS_INDEX: {
+      name: ScreenName.Index,
+      params: {
+        title,
+        query: QUERY_TYPES.EVENT_RECORDS,
+        queryVariables: { limit: 15, order: 'listDate_ASC' },
+        rootRouteName: ROOT_ROUTE_NAMES.EVENT_RECORDS
+      }
+    },
+    NEWS_ITEMS_INDEX: ({
+      categoryId,
+      categoryTitle,
+      categoryTitleDetail,
+      indexCategoryIds,
+      rootRouteName
```

Comment:
what about setting the fallback here `rootRouteName = ROOT_ROUTE_NAMES.NEWS_ITEMS` and have just `rootRouteName` in line 82?

Reply:
How about adding `rootRouteName` into the `data` array? I suppose it would be more useful? What do you think?

Reply:
I thought it would be enough to update only the data array in case of a future change. `ROOT_ROUTE_NAMES` comes from `consts`.

Reply:
yes, that's right 🤦‍♂️